sites/all/modules/hr/hr_events/js/hr_events.js

```diff
@@ -0,0 +1,16 @@
+
+(function($) {
+  Drupal.behaviors.hrEventsDatePopupValue = {
+    attach: function (context, settings) {
+      $('#edit-field-event-date-und-0-value-datepicker-popup-0', context).on('change', function(){ 
+          $('#edit-field-event-date-und-0-value2-datepicker-popup-0').val($(this).val());
+      });
+<<<<<<< HEAD
+      $('#edit-field-event-date-und-0-value-timeEntry-popup-1', context).on('change', function(){ 
+          $('#edit-field-event-date-und-0-value2-timeEntry-popup-1').val($(this).val());
+      });
+=======
+>>>>>>> 74ff28a2f8e5ae55e3466fb8d8b038bac594de4a
```

Comment:
@pcambra I was fixing a git confict, which was resolved by adding the changed file to the staging area, committing and pushing it to the repo.